src/components/date-picker.tsx

```diff
@@ -0,0 +1,88 @@
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import Button from '@src/components/button';
+import tamaguiConfig from '@src/tamagui';
+import { useState } from 'react';
+import { Platform } from 'react-native';
+import { H5, Sheet, YStack, Text } from 'tamagui';
+
+type Props = {
+    getDate: (date: Date) => void;
+};
+
+export const DatePicker = ({ getDate }: Props) => {
+    const [date, setDate] = useState<Date>(new Date());
+    const [displayDate, setDisplayDate] = useState<string | null>();
+    const [show, setShow] = useState(false);
+
+    const onChange = (_event: any, selectedDate: any) => {
+        const currentDate = selectedDate;
+        setDate(currentDate);
+        setDisplayDate(currentDate.toISOString().slice(0, 10).replace(/-/g, ' - '));
+    };
+
+    const showDatepicker = () => {
+        setShow(true);
+    };
+
+    const closeDatePicker = () => {
+        setShow(false);
+        getDate(date);
+    };
+
+    return (
+        <>
+            <Button
+                secondary
+                h="$5"
+                icon={
+                    <MaterialCommunityIcons
+                        size={27}
+                        name="calendar"
+                    />
+                }
+                onPress={showDatepicker}>
+                <Text
+                    color={tamaguiConfig.tokens.color.primary.val}
+                    fontWeight="bold"
+                    fontSize={14}>
```

Comment:
Font size should be derived from the tamagui itself. In fact 14px is the default value either. So no need for it